File: INFRA/style/themes/python/eig_tokens.py
# Derived from INFRA/style/tokens/eig-style-tokens.v1.json.
# Keep in sync with that file when tokens change.

TOKENS_VERSION = "1.0.0"
TOKENS_GENERATED_ON = "2026-02-15"

BRAND_COLORS = {
  "eig_black": "#000000",
  "eig_blue_200": "#B3D6DD",
  "eig_blue_800": "#194F8B",
  "eig_cream_100": "#FEECD6",
  "eig_cyan_700": "#176F96",
  "eig_gold_600": "#E1AD28",
  "eig_green_500": "#5E9C86",
  "eig_green_700": "#19644D",
  "eig_purple_800": "#39274F",
  "eig_tan_300": "#F0B799",
  "eig_tan_500": "#D6936F",
  "eig_teal_900": "#024140",
  "eig_white": "#FFFFFF"
}
BRAND_RGB = {
  "eig_black": [
    0,
    0,
    0
  ],
  "eig_blue_200": [
    179,
    214,
    221
  ],
  "eig_blue_800": [
    25,
    79,
    139
  ],
  "eig_cream_100": [
    254,
    236,
    214
  ],
  "eig_cyan_700": [
    23,
    111,
    150
  ],
  "eig_gold_600": [
    225,
    173,
    40
  ],
  "eig_green_500": [
    94,
    156,
    134
  ],
  "eig_green_700": [
    26,
    101,
    77
  ],
  "eig_purple_800": [
    57,
    39,
    79
  ],
  "eig_tan_300": [
    240,
    183,
    153
  ],
  "eig_tan_500": [
    214,
    147,
    111
  ],
  "eig_teal_900": [
    4,
    65,
    64
  ],
  "eig_white": [
    255,
    255,
    255
  ]
}
BRAND_STATUS = {
  "eig_black": "provisional",
  "eig_blue_200": "primary",
  "eig_blue_800": "primary",
  "eig_cream_100": "primary",
  "eig_cyan_700": "primary",
  "eig_gold_600": "primary",
  "eig_green_500": "primary",
  "eig_green_700": "primary",
  "eig_purple_800": "primary",
  "eig_tan_300": "primary",
  "eig_tan_500": "primary",
  "eig_teal_900": "primary",
  "eig_white": "primary"
}
PROVISIONAL_TOKEN_IDS = [
  "eig_black"
]

EIG_DISCRETE = [
  "#19644D",
  "#E1AD28",
  "#176F96",
  "#D6936F",
  "#194F8B",
  "#5E9C86",
  "#39274F"
]
SEQUENTIAL = {
  "eig_seq_blue_5": [
    "#164C87",
    "#236BB7",
    "#419EF1",
    "#79C5FD",
    "#C1DCFD"
  ],
  "eig_seq_red_5": [
    "#97310E",
    "#D34917",
    "#F97D1E",
    "#FBA94F",
    "#FDD7A8"
  ]
}
LEGACY_SEMANTIC_SETS = {
  "dci_quintile": {
    "at_risk": "#F97D1E",
    "comfortable": "#419EF1",
    "distressed": "#97310E",
    "mid_tier": "#9FD2C0",
    "prosperous": "#164C87"
  },
  "educ_attainment": {
    "bachelors_plus": "#164C87",
    "hs_diploma": "#FDD7A8",
    "no_hs_diploma": "#F97D1E",
    "some_college": "#C1DCFD"
  },
  "occupation": {
    "natural_resources_and_construction": "#FBA94F",
    "production": "#776FD6",
    "professional": "#FFC0CB",
    "sales": "#79C5FD",
    "services": "#008080"
  },
  "race": {
    "asian": "#236BB7",
    "black": "#FBA94F",
    "hispanic": "#776FD6",
    "native_american": "#DD88CF",
    "white": "#79C5FD"
  },
  "urban_rural": {
    "rural": "#274653",
    "small_town": "#008080",
    "suburban": "#254F85",
    "urban": "#79C5FD"
  }
}

TYPOGRAPHY = {
  "body": {
    "fallback_stack": [
      "Arial",
      "Helvetica Neue",
      "Helvetica",
      "sans-serif"
    ],
    "primary_family": "Galaxie Polaris",
    "source": "2022",
    "status": "primary"
  },
  "headline": {
    "fallback_stack": [
      "Tiempos Text",
      "Source Serif 3",
      "Georgia",
      "Times New Roman",
      "serif"
    ],
    "primary_family": "Tiempos Headline",
    "source": "2022",
    "status": "primary"
  },
  "legacy_families": [
    {
      "family": "Galaxie Polaris",
      "source": "2020",
      "status": "legacy"
    },
    {
      "family": "Tiempos Text",
      "source": "2020",
      "status": "legacy"
    }
  ],
  "roles": {
    "axis_legend_table_body": {
      "family": "Galaxie Polaris",
      "size_px_range": [
        9,
        10
      ],
      "source": "2022",
      "status": "primary",
      "weight": "regular"
    },
    "chart_title": {
      "family": "Tiempos Headline",
      "size_px_range": [
        14,
        16
      ],
      "source": "2022",
      "status": "primary",
      "weight": "semibold"
    },
    "notes_caption": {
      "family": "Galaxie Polaris",
      "size_px": 8,
      "source": "2022",
      "status": "primary",
      "weight": "regular"
    },
    "subtitle": {
      "family": "Galaxie Polaris",
      "size_px_range": [
        11,
        12
      ],
      "source": "2022",
      "status": "primary",
      "weight": "semibold"
    }
  }
}
# Legacy figure size (2020 standard). No current-standard replacement is defined in
# the token system yet. See INFRA/docs/eig-figure-style.md for chart layout guidance.
SIZING = {
  "figure_default": {
    "height_in": 3.5,
    "source": "2020",
    "status": "legacy",
    "width_in": 6.5
  }
}
POLICIES = {
  "legacy_palette_decision_rule": [
    "Use primary (2022) palette by default for all new outputs.",
    "Use legacy semantic sets only when preserving longitudinal comparability with prior published EIG series or contractual client requirements.",
    "If legacy semantic sets are used, annotate output metadata with `legacy_palette_justification` and `legacy_set_id`.",
    "Do not mix primary categorical palette and legacy semantic set within one legend unless dual-encoding is explicitly required and documented."
  ]
}

File: INFRA/style/themes/r/eig_tokens.R
# Derived from INFRA/style/tokens/eig-style-tokens.v1.json.
# Keep in sync with that file when tokens change.

EIG_TOKENS_VERSION <- "1.0.0"
EIG_TOKENS_GENERATED_ON <- "2026-02-15"

EIG_COLORS <- c(
  "eig_teal_900" = "#024140",
  "eig_green_700" = "#19644D",
  "eig_green_500" = "#5E9C86",
  "eig_blue_800" = "#194F8B",
  "eig_blue_200" = "#B3D6DD",
  "eig_cream_100" = "#FEECD6",
  "eig_purple_800" = "#39274F",
  "eig_gold_600" = "#E1AD28",
  "eig_cyan_700" = "#176F96",
  "eig_tan_500" = "#D6936F",
  "eig_tan_300" = "#F0B799",
  "eig_black" = "#000000",
  "eig_white" = "#FFFFFF"
)

EIG_DISCRETE <- c("#19644D", "#E1AD28", "#176F96", "#D6936F", "#194F8B", "#5E9C86", "#39274F")

EIG_PROVISIONAL_TOKEN_IDS <- c("eig_black")

EIG_SEQUENTIAL <- list(
  "eig_seq_blue_5" = c("#164C87", "#236BB7", "#419EF1", "#79C5FD", "#C1DCFD"),
  "eig_seq_red_5" = c("#97310E", "#D34917", "#F97D1E", "#FBA94F", "#FDD7A8")
)

EIG_LEGACY_SEMANTIC_SETS <- list(
  "dci_quintile" = list(
      "distressed" = "#97310E",
      "at_risk" = "#F97D1E",
      "mid_tier" = "#9FD2C0",
      "comfortable" = "#419EF1",
      "prosperous" = "#164C87"
    ),
  "educ_attainment" = list(
      "no_hs_diploma" = "#F97D1E",
      "hs_diploma" = "#FDD7A8",
      "some_college" = "#C1DCFD",
      "bachelors_plus" = "#164C87"
    ),
  "urban_rural" = list(
      "suburban" = "#254F85",
      "urban" = "#79C5FD",
      "small_town" = "#008080",
      "rural" = "#274653"
    ),
  "occupation" = list(
      "professional" = "#FFC0CB",
      "sales" = "#79C5FD",
      "services" = "#008080",
      "production" = "#776FD6",
      "natural_resources_and_construction" = "#FBA94F"
    ),
  "race" = list(
      "black" = "#FBA94F",
      "white" = "#79C5FD",
      "hispanic" = "#776FD6",
      "asian" = "#236BB7",
      "native_american" = "#DD88CF"
    )
)

EIG_FONT_HEADLINE_PRIMARY <- "Tiempos Headline"
EIG_FONT_BODY_PRIMARY <- "Galaxie Polaris"

# Legacy figure size (2020 standard). No current-standard replacement is defined in
# the token system yet. See INFRA/docs/eig-figure-style.md for chart layout guidance.
EIG_FIGURE_DEFAULT_WIDTH_IN <- 6.5
EIG_FIGURE_DEFAULT_HEIGHT_IN <- 3.5

EIG_LEGACY_PALETTE_DECISION_RULE <- c("Use primary (2022) palette by default for all new outputs.", "Use legacy semantic sets only when preserving longitudinal comparability with prior published EIG series or contractual client requirements.", "If legacy semantic sets are used, annotate output metadata with `legacy_palette_justification` and `legacy_set_id`.", "Do not mix primary categorical palette and legacy semantic set within one legend unless dual-encoding is explicitly required and documented.")

